cni/cmd/istio-cni-repair/main.go

```diff
@@ -166,6 +176,12 @@ func logCurrentOptions(bpr *repair.BrokenPodReconciler, options *ControllerOptio
 	}
 }
 
+func init() {
```

Comment:
You don't need to init here, metric be inited in the package where it defines.

Reply:
I think this is placed in init() so that all metrics for the package can be stored in the metric struct.  @stewartbutler can you clarify?

Reply:
Yes, that was the goal. That way you could pass the metric struct to the reconciler and have a guarantee that it was initialized. I'm not sure if that requirement is changed now that you are using istio.io/pkg/monitoring though.

Reply:
The metric could just live and init in other package. As long as the package is imported directly or indirectly by the main package, the init function there will be triggered.